.storybook/src/GeoModel-layer.stories.ts

```diff
@@ -0,0 +1,179 @@
+import GeoModelLayer from '../../src/layers/GeoModelLayer';
+import { scaleLinear } from 'd3-scale';
+import { GeoModelData } from '../../src/interfaces';
+
+export default {
+  title: 'PIXI JS WebGL Layer',
+};
+
+const width = 400;
+const height = 500;
+
+const xbounds = [0, 1000];
+const ybounds = [0, 1000];
+
+export const GeoModel = () => {
+  const geoModelLayer = new GeoModelLayer('webgl', {});
+
+  const root = document.createElement('div');
+  root.className = 'grid-container';
+  root.setAttribute(
+    'style',
+    `height: ${height}px; width: ${width}px;background-color: #eee;`,
+  );
+  root.setAttribute('height', `${height}`);
+  root.setAttribute('width', `${width}`);
+
+  geoModelLayer.onMount({ elm: root, height, width });
+  geoModelLayer.onUpdate(createEventObj(root));
+
+  return root;
+};
+
+const createEventObj = (elm: any) => {
+  const xscale = scaleLinear()
+    .domain(xbounds)
+    .range([0, width]);
+  const yscale = scaleLinear()
+    .domain(ybounds)
+    .range([0, height]);
+  const data: GeoModelData[] = [
```

Comment:
For the data format we're using in REP there is one array of values for x, one array for top and one for bottom. The reason for this is that the surfaces are sampled along points on the wellbore in 2D on the map. The same points are used for all request so there should be a 1:1 between points on the wellbore and depths of surfaces. I'm not sure if there will be other data sources which would use a different format. Maybe it would then make sense to have both a top and bottom pos in additon to depth, but for our case we don't need it. As far as I can see the second coordinate for the pos is never used? If so it should be removed. I think using 'md' is confusing as this is normally measured depth along the wellbore. Maybe we could use TVD instead?